src/main/java/org/myrobotlab/service/config/OakDConfig.java

```diff
@@ -0,0 +1,32 @@
+package org.myrobotlab.service.config;
+
+import org.myrobotlab.framework.Plan;
+
+public class OakDConfig extends ServiceConfig {
+
+  
+  /**
+   * install through py4j
+   */
+  public boolean py4jInstall = true;
```

Comment:
why does this need to be part of the config?  setting this to false is just not valid...

Reply:
I think @AutonomicPerfectionist added it ...
It is valid in the sense that Py4j service java-land can connect to a non-self installed python instance if the Py4j.py script is run.

Reply:
I did not add that; my method of python services doesn't use py4j nor does it use really any Java-side per-service components, it's all handled in Python land

Reply:
oop, thought this was in the Py4j service context ...
yeah, this is starting to think about how Py4j & OakD would work with single click ...
The OakD demo installs were non trivial ... 
Non trivial in that, If I remember they did a lot more than just install a pip module ...
https://docs.luxonis.com/en/latest/pages/tutorials/first_steps/ 
I'm not sure how we want to handle it .. from a consumer's point of view the goal almost always is ... I click worky
Getting it to work that way makes me think docker is the way to go ... :shrug:

Reply:
Looking through the code, this looks much more akin to my first attempt at Python services, where each service needed a Java-side proxy. I think focusing efforts on the new system I've designed would be a better approach, as adding a new Python service would not involve duplicating method definitions between Java and Python, and python services are allowed to exist separately from a singular gateway implementation (nothing stops mrlpy from supporting py4j as a gateway, but it also works across websockets and potentially other gateways)

Reply:
> Non trivial in that, If I remember they did a lot more than just install a pip module ...
https://docs.luxonis.com/en/latest/pages/tutorials/first_steps/

Oh yeah, we definitely want Docker for that

Reply:
Yeah .. docker .. but that means making a Docker service in java land if we want to use it there ...
it also means adding infrastructure e.g. Docker registry account etc ... 
As regards to the Java-Land proxy vs Python stand alone service I totally agree and don't want required Java proxies to exists for Python.  I'd prefer it be network only requirements.  
It's trivial to have a python client with mrl .. but there currently is no service framework within Python that can mirror a Java-land instance - until that happens I don't believe a python only service is an option.

Reply:
My mrlpy project can, and has had the ability for a while. You can write Python services with it right now, and it will autogenerate python-land proxies for Java services. The only missing part is allowing a user to start a Python service from the WebGui, which means that Java-land needs the capacity to start the Python-side runtime. That's what #1341 is doing.

An example of this is here:

https://github.com/AutonomicPerfectionist/mrlpy/blob/develop/examples/llama-with-solr.py

That particular example won't work with the latest MRL because it uses service interfaces that haven't been merged, but simpler examples do work out of the box (or at least did last I checked a few weeks ago)